Project-Hotpot/Project-Hotpot/PFSong.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  PFSong.swift
+//  Project-Hotpot
+//
+//  Created by Harini Sundaram on 7/14/22.
+//
+
+import UIKit
+import Parse
+
+//will store Songs as PFSong objects
+class PFSong: PFObject, PFSubclassing {
+    @NSManaged var name : String
+    @NSManaged var URI: String
+    @NSManaged var duration: Int
+    @NSManaged var artist: String
+    @NSManaged var album: String
+    
+    static func parseClassName() -> String {
+        return "Song"
+    }
```

Comment:
What is the purpose of this?

Reply:
This is a mandatory class function by Parse. I have to implement it, or else it wouldn't be allow me to make a Parse Class Object